bolt-modules/boltlib/spec/functions/add_facts_spec.rb

```diff
@@ -14,6 +14,7 @@
   around(:each) do |example|
     Puppet[:tasks] = tasks_enabled
     Puppet.override(bolt_executor: executor, bolt_inventory: inventory) do
+      inventory.stubs(:version).returns(1)
```

Comment:
For now most stdlib tests are restricted to V1 target/inventory. Shall I duplicate/add conditions for v2? or just replace all together?

Reply:
I wonder if we're better off testing these in `spec/pal`

Reply:
@puppetlabs/bolt Open question

Reply:
Lets ticket this separately or add it as a requirement to the merge inventory versions work

Reply:
https://github.com/puppetlabs/bolt/issues/1257